NEWS.rst

```diff
@@ -2,6 +2,45 @@
 NEWS
 ******
 
+1.2.8
+=====
+
+::
```

Comment:
Might be worth pointing out somewhere this is the last release of v1.2

Reply:
I think we can mention that in the slack announcement